pages/_includes/StructureDefinition-qicore-adverseevent-introduction.html

```diff
@@ -0,0 +1,3 @@
+<tr>
+    <td>See the <a href="http://hl7.org/fhir/us/qicore/patterns.html">patterns</a> page for implementation and usage patterns.</td>
```

Comment:
This path should be relative ("patterns.html"). Same applies to all the introduction content.